app/controllers/home_controller.rb

```diff
@@ -1,5 +1,7 @@
 class HomeController < ApplicationController
   def index
+    @picked_up_pets = PickedUpPet.includes(:pet).order(picked_up_at: :desc).to_a.map(&:pet)
```

Comment:
配列に直してしまうと余分な計算が発生してしまうので、出力したいものをベースのモデルに持ってくると良さそうです。

```suggestion
    @picked_up_pets = Pet.joins(:picked_up_pets).order(picked_up_pets: { picked_up_at: :desc })
```
※ 動作確認していないのでこのまま動くかわからないです；

joins で繋げることで双方のテーブルにないものは弾かれるので、ピックアップされたペットだけを取得できる様になります。